runtime/jcl/j9jcl.tdf

```diff
@@ -702,3 +702,6 @@ TraceExit=Trc_JCL_J9SigUsr2Startup_Exit noEnv Overhead=1 Level=1 Template="J9Sig
 
 TraceEntry=Trc_JCL_J9SigUsr2Shutdown_Entry noEnv Overhead=1 Level=1 Template="J9SigUsr2Shutdown"
 TraceExit=Trc_JCL_J9SigUsr2Shutdown_Exit noEnv Overhead=1 Level=1 Template="J9SigUsr2Shutdown"
+
+TraceException=Trc_JCL_memoryManagement_verifyMemoryUsageAfterGC_memoryUsageError Test noEnv Overhead=1 Level=1 Template="JCL: verifyMemoryUsageAfterGC memoryUsageError: GcName=%s, MemoryPool=%s, initialSize=%lld, preUsedSize=%llu, preCommittedSize==%llu, preMaxSize=%lld, postUsedSize=%llu, postCommittedSize==%llu, postMaxSize=%lld"
+TraceEvent=Trc_JCL_memoryManagement_verifyMemoryUsageAfterGC_memoryUsage Test noEnv Overhead=1 Level=6 Template="JCL: verifyMemoryUsageAfterGC memoryUsage: GcName=%s, MemoryPool=%s, initialSize=%lld, preUsedSize=%llu, preCommittedSize==%llu, preMaxSize=%lld, postUsedSize=%llu, postCommittedSize==%llu, postMaxSize=%lld"
```

Comment:
Even though it's not case-sensitive, please use `NoEnv` instead of `noEnv` (see [TDFParser.cpp](https://github.com/eclipse/omr/blob/master/tools/tracegen/TDFParser.cpp#L220)).